### PR TITLE
Move Loyalty Button

### PR DESCRIPTION
**Pull request by r-cochran**
### Summary
Moved the Customer Loyalty button from the sale total panel to a sub header section.

### Screenshots
Customer unlinked
image

Customer linked
image

### CODE REVIEW

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/core/services/icon.service.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ export class IconService {
         private iconRegistry: MatIconRegistry) {
 
         this.icons.set('Account', ['account_balance.svg', this.defaultIconProvider]);
+        this.icons.set('AccountCircle', ['account_circle.svg', this.defaultIconProvider]);
```
Comment (dankaste) [inline openpos-client-libs/projects/openpos-client-core-lib/src/lib/core/services/icon.service.ts:24]: We actually don't need this anymore. It should work to just send over the icon key `account_circle`.

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/screens-with-parts/sale/sale.component.ts [hunk 2]
```diff
@@ -74,6 +74,10 @@ export class SaleComponent extends PosScreen<SaleInterface> {
         }
     }
 
+    public doMenuItemAction(menuItem: IActionItem) {
```
Comment (dankaste) [inline openpos-client-libs/projects/openpos-client-core-lib/src/lib/screens-with-parts/sale/sale.component.ts:77]: you should be able to just call doAction from the html template

**Comment by chenson42** [comment]
This needs to be configurable.   Another customer wants the original layout.

**Comment by chenson42** [comment]
I think the button should be the same height prior to and after the custom is linked

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/components/accent-button/accent-button.component.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  &.mobile {
+    border-width: 1px;
+    font-size: $text-xs-mobile*1.25;
+    min-width: 75px;
+    padding: $text-sm-mobile/8 $text-sm-mobile*.75;
+  }
+}
+
+:host ::ng-deep app-icon {
+  margin: -4px -4px -4px 0;
```
Comment (dankaste) [inline openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/components/accent-button/accent-button.component.scss:29]: if we can avoid ng-deep that would be best. Last I checked angular announced its plans to deprecate this.

**Review by dankaste — COMMENTED** [review]
This is just personal preference but I like the button on the left similar to how it looks in the mobile view.

**Comment by mmichalek** [comment]
This is a more precise mock per Chris' comment and showing the JM colors. 
image

image

**Comment by chenson42** [comment]
> This is just personal preference but I like the button on the left similar to how it looks in the mobile view.

Yeah.  This was the compromise between Petco and AEO.

**Review by chenson42 — APPROVED** [review]

**Comment by r-cochran** [comment]
latest change included making the Loyalty ID label i18n based instead of static

**Pull request merged**